### ISSUE TITLE
Width on sticky component not updating on resize

### ISSUE THREAD

**Issue by joshkg** [kaliber5/ember-sticky-element#17]
I can see that the component is setting the width in an inline style -- however if I resize my browser (shrinking) the width is not updating along with it, breaking layout.

Can we add an option to set the width (in my case, 100% would do it) or at least throw a classNames property on the component so I can set it via CSS? Thanks! Great addon!

**Comment by skyroot1022** [comment]
I've same error.

**Comment by skyroot1022** [comment]
Hi joshkg.
Did you solve this problem?

**Comment by joshkg** [comment]
@skyroot1022 Sort of :) If you wrap your {{sticky-element}} component in the HBS with another element that has a class name, you can override the inline width with CSS. Its brittle though, and only works if your sticky element can use percentage or other non-changing width value.

**HBS**
```
<div class="sticky-element-wrapper>
	{{sticky-element}}
</div>
```
**CSS**
```
.sticky-element-wrapper > div, .sticky-element {
	width: 100% !important;
} 
```

**Comment by simonihmig** [comment]
so,
1. being able to set a custom class would be a useful feature. Would just require putting a `{{class}}`  item into the existing classes list here: https://github.com/kaliber5/ember-sticky-element/blob/master/addon/templates/components/sticky-element.hbs#L2. Anyone interested in contributing a PR? :)
2. I am not sure how setting the width to 100% would help here. In sticky mode, it has `position: fixed`, and `width: 100%` would mean the whole width of the page (not that of the parent). It should always inherit the width of the parent though. Or do I miss something?
3. Another solution would be to listen for `resize` events and update the width accordingly

**Comment by joshkg** [comment]
@simonihmig Regarding custom class, I was thinking even more simply just adding a `classNames: ['sticky-element-container']` or such on the component context -- inline width style gets set on the component's tag, so to address my issue I would need to be able to select the whole component via CSS. I wasn't suggesting giving the component a width of 100% for all uses, I was just noting that, in my 1 use case, that's what I need, which is addressable if I can select the component via CSS.

**Comment by simonihmig** [comment]
@joshkg Ok, got it! Would you be able to submit a PR? :)

**Comment by joshkg** [comment]
Done! Do you want to call it part of 0.1.5 or bump?

**Comment by simonihmig** [comment]
@joshkg Published as 0.1.6!

**Comment by joshkg** [comment]
Thanks!
